packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.styles.ts

```diff
@@ -176,28 +176,52 @@ export const getStyles = memoizeFunction(
       labelDisabled: {},
       spinButtonWrapper: {
         display: 'flex',
+        position: 'relative',
         boxSizing: 'border-box',
         height: DEFAULT_HEIGHT,
         minWidth: DEFAULT_MIN_WIDTH,
-        border: `1px solid ${SpinButtonRootBorderColor}`,
-        borderRadius: effects.roundedCorner2
+        selectors: {
+          ':after': {
+            pointerEvents: 'none',
+            content: "''",
+            position: 'absolute',
+            left: 0,
+            top: 0,
+            bottom: 0,
+            right: 0,
+            border: `1px solid ${SpinButtonRootBorderColor}`,
+            borderRadius: effects.roundedCorner2
+          }
+        }
       },
       spinButtonWrapperTopBottom: {
         width: '100%'
       },
       spinButtonWrapperHovered: {
-        borderColor: SpinButtonRootBorderColorHovered,
         selectors: {
+          ':after': {
+            borderColor: SpinButtonRootBorderColorHovered
+          },
           [HighContrastSelector]: {
             borderColor: 'Highlight'
           }
         }
       },
       spinButtonWrapperFocused: {
-        borderColor: SpinButtonRootBorderColorFocused,
         selectors: {
           [HighContrastSelector]: {
             borderColor: 'Highlight'
+          },
+          ':after': {
+            pointerEvents: 'none',
+            content: "''",
+            position: 'absolute',
+            left: -1,
+            top: -1,
+            bottom: -1,
+            right: -1,
+            border: `2px solid ${SpinButtonRootBorderColorFocused}`,

```

Comment:
Do you need a high contrast color for this too?

Reply:
it's there.